spring-cloud-starter-stream-source-websocket/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>spring-cloud-starter-stream-source-websocket</artifactId>
+	<packaging>jar</packaging>
+	<name>spring-cloud-starter-stream-source-websocket</name>
+	<description>Spring Cloud Stream websocket-source core</description>
+
+	<parent>
+		<groupId>org.springframework.cloud.stream.app</groupId>
+		<artifactId>websocket-app-starters-build</artifactId>
+		<version>2.0.0.BUILD-SNAPSHOT</version>
+	</parent>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-websocket</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-websocket</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-starter-stream-kafka</artifactId>
```

Comment:
Kafka? No, no Kafka as a dependency in this kind of project.
There is a `TestBinder` picked up automatically by Spring Boot.
But what I see you just don't have any auto-configuration detection in your test-case.
I think the `@SpringBootApplication` what we have in the `WebsocketSinkIntegrationTests` is the right way to go.